src/librustc_resolve/lib.rs

```diff
@@ -730,6 +741,14 @@ enum UseLexicalScopeFlag {
     UseLexicalScope,
 }
 
+#[allow(dead_code)]
+enum LookupTarget {
+    // Lookup only in the current crate
+    CurrentCrate,
```

Comment:
Good question. Originally I was looking only in the current crate, but it makes more sense to always look everywhere.
Should I remove it?